builtin/aws/ecs/plugin.proto

```diff
@@ -11,7 +11,14 @@ message Deployment {
   string task_arn = 2;
   string service_arn = 3;
   string target_group_arn = 4;
-  string load_balancer_arn = 5;
+
+  // Reference to the load balancer. Either the LB itself (if we created it),
+  // or the lb listener if the user specified it.
+  oneof lb_reference {
+    string load_balancer_arn = 5;
+    string listener_arn = 8;
+  }
```

Comment:
This looks like it could be a breaking protobuf change, but I don't think it is! It's not changing field numbers. I also tried releasing a deployment made with the previous compiled protobufs, and it worked.